packages/contentful/src/crn/queries/discover.queries.ts

```diff
@@ -0,0 +1,96 @@
+/* istanbul ignore file */
+
+import { gql } from 'graphql-tag';
+import { pagesContentQueryFragment } from './pages.queries';
+import { tutorialsContentQueryFragment } from './tutorials.queries';
+
+const usersContentQueryFragment = gql`
+  fragment PartialUsersContent on Users {
+    sys {
+      id
+      firstPublishedAt
+      publishedAt
+      publishedAt
+      publishedVersion
+    }
+
+    alumniSinceDate
+    createdDate
+    degree
+    email
+    firstName
+    institution
+    jobTitle
+    lastName
+    onboarded
+    role
```

Comment:
looking at the query from crn (squidex) it seems that we're not retrieving some of these fields
```suggestion
    email
    firstName
    institution
    jobTitle
    lastName
```

Reply:
Yeah, I got thrown off by [the unit test fixtures](https://github.com/yldio/asap-hub/blob/master/apps/crn-server/test/fixtures/discover.fixtures.ts#L117-L177) defining _everything_.

I've removed the extra properties now.